theme/ItaliaTheme/Blocks/_iconBlocks.scss

```diff
@@ -47,6 +47,13 @@
     }
   }
   .container {
+    &.secondary,
+    &.transparent {
+      .title,
+      .description {
+        color: $body-color;
```

Comment:
Usa `$secondary-text`, così in siti in cui il secondario è più scuro avranno il testo chiaro, se le variabili sono impostate bene